database/migrations/2016_06_01_000001_create_oauth_auth_codes_table.php

```diff
@@ -12,9 +12,9 @@
     public function up(): void
     {
         Schema::create('oauth_auth_codes', function (Blueprint $table) {
-            $table->string('id', 100)->primary();
-            $table->unsignedBigInteger('user_id')->index();
-            $table->unsignedBigInteger('client_id');
+            $table->char('id', 80)->primary();
+            $table->foreignId('user_id');
```

Comment:
Why no index here anymore?

Reply:
Because the column itself is redundant #1752